integration-tests/test_client_options.py

```diff
@@ -177,3 +179,38 @@ def test_check_show_results(insights_client):
         assert "OPENSSH_HARDENING_CONFIG_PERMS" in show_results.stdout
     finally:
         os.chmod("/etc/ssh/sshd_config", 0o600)
+
+
+def test_common_specs(insights_client):
```

Comment:
We need to be careful with grouping of tests. This test should not be in test_client_options.py because its not validating any insights-client options, please move it to a new file test_common_specs.py.

Reply:
done